client/src/components/ReviewForm.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+
+const ReviewForm = ({ postReview }) => {
+    const [newReview, setNewReview] = useState({
+        body: ""
+    })
+
+    const handleChange = (event) => {
+        setNewReview({
+            ...newReview,
+            [event.target.name]: event.target.value
+        })
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        postReview(newReview)
+        clearForm()
+    }
+
+    const clearForm = () => {
+        setNewReview({
+            body: ""
+        })
+    }
+
+    return (
+        <>
+            
+            <div className="review-tile callout">
```

Comment:
extra space, and the fragment should be removed because this single div is the top level element